src/OracleDatabase/OracleDatabase.Autorest/examples/Get-AzOracleDatabaseAutonomousDatabase.md

```diff
@@ -0,0 +1,352 @@
+### Example 1: Gets a list of the Autonomous Database resources
+```powershell
+Get-AzOracleDatabaseAutonomousDatabase
+```
+
+```output
+Location           Name                       SystemDataCreatedAt SystemDataCreatedBy                      SystemDataCreatedByType SystemDataLastModifiedAt SystemDataLastModifiedBy             SystemDataLastModifiedByType ResourceGroupName
+--------           ----                       ------------------- -------------------                      ----------------------- ------------------------ ------------------------             ---------------------------- -----------------
+eastus             ADBScli1                   05/07/2024 13:07:02 ramakrishnan.vilathur.sriniva@oracle.com User                    05/07/2024 13:15:49      857ad006-4380-4712-ba4c-22f7c64d84e7 Application                  SDKTestRG
```

Comment:
You can manually update the output and make it shorter.
Don't forget to mask your mails. You can use a fake mail instead.

Reply:
Updated